booking-engine-widget/integrations/google-triggers-reference.md

```diff
@@ -336,6 +341,24 @@ We are pushing all available variables into the eCommerce tracking for more adva
       adults: 2, // Count of adults in  reservation
       children: 0 // Count of children in reservation
     }
+    {
+      item_id: "ec34da8b-0932-493e-a742-baf1af194789", // Product id
```

Comment:
There's no 3rd item type for upsell promoted services. We already serialize them as a separate reservations.

Reply:
It's good like this